lib/active_model_serializers/adapter/json_api/relationship.rb

```diff
@@ -34,6 +34,11 @@ def as_json
 
         private
 
+        def ensure_valid!(json)
```

Comment:
That's sortof where I was heading. As far as I can see, the actual spec file does not list any required keys for `relationships`. So I'm curious to know where the requirement for one of `data`, `links`, or `meta` stems from.